src/components/ViewRequest/Actions/ReworkRequest.tsx

```diff
@@ -24,13 +24,30 @@ const ReworkRequest = () => {
 
   const updateHandler = () => {
     if (request.data && currentStage?.previous) {
-      const newStage = currentStage.previous;
-      const eventTitle = currentStage.previousEventTitle;
-      updateStage.mutate({
+      let newData = {
```

Comment:
newData is never reassigned

```suggestion
      const newData = {
```